bin/sku.js

```diff
@@ -20,10 +29,9 @@ switch (script) {
   case 'start':
   case 'start-ssr':
   case 'storybook':
-  case 'chromatic':
-  case 'configure': {
-    const scriptPath = require.resolve(`../scripts/${script}`);
```

Comment:
Any idea why this was previously doing `require.resolve`? Wary of breaking a use case we're unaware of.